hack/test-e2e.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#shellcheck disable=SC2086
+
+set -e
+
+image="origin-libvirt-machine-controllers:latest"
```

Comment:
We should use commit based tag so in case the e2e fails we know which commit was actually tested. So at least `image="${MACHINE_CONTROLLERS_IMAGE:-origin-libvirt-machine-controllers:latest}"`